### PR TITLE
Sentence case

### PR DESCRIPTION
**Pull request by RobLoweMOJ**
Labels are sentence case
Role type options keeps upper case first letter
Working pattern options now in sentence case

### CODE REVIEW

File: web/app/themes/justicejobs/search.php [hunk 1]
```diff
@@ -100,34 +100,34 @@ class="hero__img hero__img--mobile"
...
+                <span class="filter__label">Working pattern</span>
                 <div class="select-list">
                     <?php
                     $terms = get_terms(array(
                         'taxonomy' => 'working_pattern',
                         'hide_empty' => true,
                     ));
 
-                    $options = jj_select_options($_working_pattern, 'Working Pattern');
+                    $options = jj_select_options($_working_pattern, 'working pattern', true);
```
Comment (wilson1000-MoJ) [inline web/app/themes/justicejobs/search.php:128]: This is the only function call with sentence_case defined as true - is this intended?

**Review by wilson1000-MoJ — COMMENTED** [review]
This is simply a review - no test has taken place however I can see inconsistencies in the jj_select_options function calls that may impact what this PR is authored to do.

File: web/app/themes/justicejobs/search.php (hunk 1 above)
Comment (RobLoweMOJ) [reply web/app/themes/justicejobs/search.php:128]: Yes. After discussing with Josh. Role type should remain Uppercase first letter for each word as they are departments. Where working patterns should be sentence case. So it is correct that this is the only place that sentence case set to true is needed for the moment.

Comment (RobLoweMOJ) [reply web/app/themes/justicejobs/search.php:128]: Also salary range is numbers so doesnt need it either

Comment (wilson1000-MoJ) [reply web/app/themes/justicejobs/search.php:128]: Ok great 👍

**Pull request merged**